lumen/sources/base.py

```diff
@@ -833,9 +832,64 @@ class BaseSQLSource(Source):
 
     load_schema = param.Boolean(default=True, doc="Whether to load the schema")
 
+    excluded_tables = param.List(default=[], doc="""
+        List of table names that should be excluded from the results.""")
```

Comment:
Either declare as constant or automatically rebuild the regex on change. Also nitpick, remove the extra empty line.